internal/scan-manager/scanners/veracode/src/main/java/org/wso2/security/tools/scanmanager/scanners/veracode/service/VeracodeScanner.java

```diff
@@ -83,10 +77,14 @@ public VeracodeScanner() throws IOException {
                 VeracodeScannerConstants.VERACODE_API_KEY));
 
         VeracodeAPIUtil.setCredentials(options);
-        CallbackUtil.setCallbackUrls(VeracodeScannerConfiguration.getInstance().getConfigProperty(ScannerConstants
-                        .SCAN_MANAGER_CALLBACK_URL) + VeracodeScannerConfiguration.getInstance().getConfigProperty(
-                ScannerConstants.SCAN_MANAGER_CALLBACK_LOG), VeracodeScannerConfiguration.getInstance()
-                        .getConfigProperty(ScannerConstants.SCAN_MANAGER_CALLBACK_URL) + VeracodeScannerConfiguration
+        CallbackUtil.setCallbackUrls(ScannerConstants.HTTP_PROTOCOL + System.getenv(ScannerConstants.SCAN_MANAGER_HOST)
```

Comment:
Can we improve this line's readability. Can we do the `Long.parseLong`, etc. separately.